server/lib/adapters/mssql.js

```diff
@@ -128,7 +128,7 @@ export function runQuery(client, query, options) {
     const conf = _.cloneDeep(client.config);
     conf.options = confoptions;
 
-    let conn = new tedious.Connection(conf);
+    let conn = client;//new tedious.Connection(conf);
```

Comment:
This is the primary fix.  We had already created a connection through the "openConnection" method.  Don't need to do it again, just use the one passed in.  closeConnection is called on it in syncagent

- runQuery

Reply:
I would remove the commented out `//new tedious.Connection(conf);` to avoid future confusion

Reply:
makes sense, i do this on first commits so that I can see the changes a little more clearly, then followup in the future by removing.  But we can remove now